pkg/client/didexchange/client.go

```diff
@@ -142,21 +142,22 @@ func (c *Client) CreateInvitationWithDID(label, did string) (*Invitation, error)
 }
 
 // HandleInvitation handle incoming invitation
-func (c *Client) HandleInvitation(invitation *Invitation) error {
+func (c *Client) HandleInvitation(invitation *Invitation) (string, error) {
```

Comment:
GoDoc comment should be updated.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/d396caf7acfa757725e07aa0bd8d5ff73bd05a84..69ccf0170928c4ab210d195c32d9105664e8944f#diff-427522b353a28b1ec350c29c79d387b0R144).